app/scripts/views/coppa/coppa-date-picker.js

```diff
@@ -90,14 +90,17 @@ define([
       if (! this._validateYear()) {
         this.addInvalidRow('#fxa-age-year');
 
-        this.showValidationError('#fxa-age-year',
-                AuthErrors.toError('YEAR_OF_BIRTH_REQUIRED'));
+        var yearError = AuthErrors.toError('YEAR_OF_BIRTH_REQUIRED');
+        this.showValidationError('#fxa-age-year', yearError);
+        this.logError(yearError);
+
       } else if (this._getYear() === CUTOFF_AGE.year &&
                ! this._validateMonthAndDate()) {
         this.addInvalidRow('#fxa-age-month, #fxa-age-date');
 
-        this.showValidationError('#fxa-age-month',
-                AuthErrors.toError('BIRTHDAY_REQUIRED'));
+        var birthdayError = AuthErrors.toError('BIRTHDAY_REQUIRED');
```

Comment:
Can you add these errors to [client-metrics.md](https://github.com/mozilla/fxa-content-server/blob/46484479951e4d784b4d517a6f520494692c760d/docs/client-metrics.md#signup)? Looks like we missed them before.